### DISCUSSION TITLE
Ciclo For

### DISCUSSION THREAD

**Discussion by AreliBernal** [LaunchX-InnovaccionVirtual/CursoIntroPython#55] [Q&A] [answered]
Hola compañeros, alguien que me pueda explicar el ciclo for de la Kata 4 Ejercicio 1, por favor.😉. Gracias.

# Ciclo for para recorrer la cadena
for sentence in text_parts:
    for key_word in key_words:
        if key_word in sentence:
            print(sentence)
            break

**Comment by miguelapol** [comment, marked as answer]
si yo puedo, mira  primero que nada usamos el doble for para que recorra los dos arreglos y luego pregunta si key_word se encuentra un elemento dentro de otro en este caso del arreglo sentence si es verdadero pues imprimira y se saldra del ciclo con el break

**Reply by AreliBernal** [reply]
Tal vez me escuche tonta pero sigo sin entender, me lo podrías explicar de diferente manera, por favor.🤔

**Reply by miguelapol** [reply]
claro mira , en key_words creamos un arreglo con palabras clave y luego usamos el tex_parts donde este divide la oración con el uso del for va ir iterando y va ir comparando palabra clave con las divisiones de la oración(text_parts).

**Reply by AreliBernal** [reply]
ok, creo que ya le entendí, muchas gracias 😁.
